### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Eionel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,20 +4,24 @@
...
 * Standalone application: no need to install anything in-cluster
+* Lens installs anywhere, elimanting the need to wrangle credentials
 * Real-time cluster state visualization
 * Resource utilization charts and trends with history powered by built-in Prometheus
-* Terminal access to nodes and containers
+* Smart terminal access to nodes and containers
+* Clusters can be local (e.g. minikube) or external (e.g. EKS, GKE, AKS)
 * Performance optimized to handle massive clusters (tested with a cluster running 25k pods)
-* Full support for Kubernetes RBAC
+* RBAC security is preserved, as Lens uses the standard kubectl API
```
Comment (nevalla) [inline README.md]: kubectl API -> Kubernetes API ?

File: README.md [hunk 2]
```diff
@@ -4,20 +4,24 @@
...
 * Real-time cluster state visualization
 * Resource utilization charts and trends with history powered by built-in Prometheus
-* Terminal access to nodes and containers
+* Smart terminal access to nodes and containers
+* Clusters can be local (e.g. minikube) or external (e.g. EKS, GKE, AKS)
 * Performance optimized to handle massive clusters (tested with a cluster running 25k pods)
-* Full support for Kubernetes RBAC
+* RBAC security is preserved, as Lens uses the standard kubectl API 
+* Lens Extensions are used to add custom visualizations and functionality to accelerate development workflows for all the technologies and services that integrate with Kubernetes
+* Helm chart service deployment: Browse and deploy Helm charts with one click-Install
```
Comment (nevalla) [inline README.md]: How about `Helm package deployment`?

**Review by Eionel — COMMENTED** [review]
Fixed the changes.

**Comment by nevalla** [comment]
Closing in favor of #1353

**Pull request closed without merging**